ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts

```diff
@@ -101,6 +101,7 @@ export interface MhRegistrationSummaryIF {
   changes?: MhRegistrationSummaryIF []
   hasDraft?: boolean
   documentId?: string
+  documentRegistrationNumber?:string
```

Comment:
We are just adding new field on top of `documentId`? Both will be used?

Reply:
They are separate fields(api returns both)... Both are used in various reports and displays and code...  In this case the MHR table child rows.  Honestly,  I am not quite sure of all the business rules around these two ids(numbers).  I know I was displaying the wrong number and it was not included in any of the interfaces...

Reply:
Ok thanks for the clarification.